index.js

```diff
@@ -8,7 +8,7 @@ var any = require('./lib/any')
 var parse = require('./lib/parse')
 
 function matches(selector, node) {
-  return Boolean(any(parse(selector), node, {one: true, shallow: true})[0])
+  return Boolean(any(parse(selector), node, {one: true, shallow: true, any})[0])
```

Comment:
```suggestion
  return Boolean(any(parse(selector), node, {one: true, shallow: true, any: any})[0])
```

Reply:
and shouldn’t this also be added to the bottom ones?

Reply:
I am not sure. To me it looks like only the two functions are affected, already updated in the PR. How do we check apart from passing tests that it does not need to be done?

Reply:
`select` and `selectAll` also should support pseudo elements. Right now only `matches` has `any`. But this is an assumption. It seems there are no tests for pseudos in `select{All}`?

Reply:
this one is I think untested and a bug

Reply:
What about https://github.com/syntax-tree/unist-util-select/blob/2269b1c4fc8dc8c824d2757612c7c398ae0da80a/test/select.js#L228 and https://github.com/syntax-tree/unist-util-select/blob/2269b1c4fc8dc8c824d2757612c7c398ae0da80a/test/select-all.js#L244

They check for pseudo element `:first-child` using `select` and `selectAll` method if I am not mistaken

Reply:
Susequently, I believe the pseudo-elements tests for select and selectAll go all the way down to L696 or L730 respectively, covering quite a lot. Not a single one tripped. Am I being mistaken here?